proxy/http/HttpTransact.cc

```diff
@@ -217,33 +217,32 @@ find_appropriate_cached_resp(HttpTransact::State *s)
 }
 
 int response_cacheable_indicated_by_cc(HTTPHdr *response);
+bool
+lt(int x, int y)
+{
+  return x < y;
+}
 
+bool
+eq(int x, int y)
+{
+  return x == y;
+}
 inline static bool
 is_negative_caching_appropriate(HttpTransact::State *s)
 {
   if (!s->txn_conf->negative_caching_enabled || !s->hdr_info.server_response.valid()) {
     return false;
   }
-
-  switch (s->hdr_info.server_response.status_get()) {
-  case HTTP_STATUS_NO_CONTENT:
-  case HTTP_STATUS_USE_PROXY:
```

Comment:
hmmm, I guess I don't understand what was wrong with the old switch. Compilers can typically optimize that into very efficient code I think.